From: MOZGIII <mike-n@narod.ru>
Date: Sat, 6 Jan 2018 04:32:03 +0300
Subject: Fixed homepage URL at setup.py

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 9098d07..a17ba08 100755
--- a/setup.py
+++ b/setup.py
@@ -9,7 +9,7 @@ setup(
     license='GPLv3',
     maintainer='Andy Grover',
     maintainer_email='agrover@redhat.com',
-    url='http://github.com/agrover/targetd',
+    url='http://github.com/open-iscsi/targetd',
     packages=['targetd'],
     scripts=['scripts/targetd']
 )
