From: MOZGIII <mike-n@narod.ru>
Date: Sat, 6 Jan 2018 04:32:14 +0300
Subject: Fixed version for 0.8.6 at setup.py

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index a17ba08..5e6d9fb 100755
--- a/setup.py
+++ b/setup.py
@@ -4,7 +4,7 @@ from distutils.core import setup
 
 setup(
     name='targetd',
-    version='0.8.5',
+    version='0.8.6',
     description='Linux remote storage API daemon',
     license='GPLv3',
     maintainer='Andy Grover',
